charts/ingress-nginx/templates/_params.tpl

```diff
@@ -63,6 +63,9 @@
 {{- if .Values.controller.disableLeaderElection }}
 - --disable-leader-election=true
 {{- end }}
+{{- if and .Values.controller.electionTTL (gt .Values.controller.electionTTL 0) }}
```

Comment:
```suggestion
{{- if .Values.controller.electionTTL }}
```

... should be fine. Otherwise you would need to cast it to int first, which is what we probably don't want to since I'd (personally) prefer a duration here.